docs/sources/clients/promtail/scraping.md

```diff
@@ -539,5 +539,5 @@ clients:
   - [ <client_option> ]
 ```
 
-Refer to [`client_config`](./configuration#client_config) from the Promtail
+Refer to [`client_config`]({{< relref "configuration.md#client_config" >}}) from the Promtail
```

Comment:
nit: Not related to your change but I'm seeing that the anchor here doesn't exist in the linked file:
```console
$ docker run -it -v "$(pwd)/docs/sources:/docs/sources" grafana/doc-validator:latest /docs/sources 2>&1 | grep -A1 docs/sources/clients/promtail/scraping.md
ERROR: /docs/sources/clients/promtail/scraping.md:542:28 The anchor "client_config" does not exist in the file "/docs/sources/clients/promtail/configuration.md".
Available anchors are: "configuring-promtail, printing-promtail-config-at-runtime, configuration-file-reference, use-environment-variables-in-the-configuration, generic-placeholders, supported-contents-and-default-values-of-configyaml, server, clients, positions, scrape_configs, pipeline_stages, docker, cri, regex, json, template, match, timestamp, output, labels, metrics, counter, gauge, histogram, tenant, replace, journal, syslog, available-labels, loki_push_api, windows_events, gcp-log, available-labels, kafka, gelf, cloudflare, heroku_drain, available-labels, relabel_configs, static_configs, file_sd_config, kubernetes_sd_config, node, service, pod, endpoints, ingress, consul_sd_config, consulagent_sd_config, docker_sd_config, limits_config, target_config, options_config, tracing_config, example-docker-config, example-static-config, example-static-config-without-targets, example-journal-config, example-syslog-config, example-push-config".
```

Reply:
I just checked it and It doesn't exits but I don't see any other anchor related to link it and fix it.

Reply:
I just took a look through the target page. I think `clients` is the heading we want as it describes an array of `<client_config>`

> # Describes how Promtail connects to multiple instances
> # of Grafana Loki, sending logs to each.
> # WARNING: If one of the remote Loki servers fails to respond or responds
> # with any error which is retryable, this will impact sending logs to any
> # other configured remote Loki servers.  Sending is done on a single thread!
> # It is generally recommended to run multiple Promtail clients in parallel
> # if you want to send to multiple remote Loki instances.
> clients:
>   - [<client_config>]